ebean-migration/src/main/java/io/ebean/migration/runner/DefaultMigrationContext.java

```diff
@@ -0,0 +1,45 @@
+package io.ebean.migration.runner;
+
+import io.ebean.migration.MigrationConfig;
+import io.ebean.migration.MigrationContext;
+
+import java.sql.Connection;
+
+/**
+ * A default implementation of the MigrationContext.
+ *
+ * @author Roland Praml, FOCONIS AG
+ */
+public class DefaultMigrationContext implements MigrationContext {
```

Comment:
It would be great, if ebean-migration can have a dependency to have access to "Database". What do you think about this idea?

Otherwise, I plan to imlement a EbeanMigrationContext extends DefaultMigrationContext, that has a `Database getDb()` method in our application and cast to this context in our JdbcMigrations

Reply:
>  have access to "Database".

Yes, I would expect it along with a ebean Transaction - so with Database and Transaction then Jdbc Migrations can use all the features of ebean etc.

Reply:
Note that I don't expect this DefaultMigrationContext to be public.